dom/src/interfaces/node.rs

```diff
@@ -40,8 +40,27 @@ pub trait NodeWrapper: sealed::Memoized + Sized {
 pub trait Child {
     /// Returns the "raw" node for this child to bind to its parent.
     fn to_bind(&self) -> &augdom::Node;
+
+    /// Identity transform
+    fn into_child(self) -> Self
+    where
+        Self: Sized,
+    {
+        self
+    }
 }
 
+/// Convert `impl std::fmt::Display` into the `impl Child`
+pub trait DisplayIntoChild: std::fmt::Display + Sized {
```

Comment:
Let's name this trait `TextChild`.

Here and below, can you import `std::fmt::Display` and `crate::text::text` at the top of the file and use `Display` and `text` here?